### PR TITLE
Chore: updates TS library and adds .tsconfig

### PR DESCRIPTION
**Pull request by ivanlori**

### CODE REVIEW

File: .tsconfig [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      "@components/*": ["components/*"],
+      "@assets/*": ["assets/*"],
+      "@layouts/*": ["layouts/*"],
+      "@pages/*": ["pages/*"],
+      "@utils/*": ["utils/*"]
+    },
+    "pretty": true
+  },
+  "esclude": ["node_modules"]
+}
```
Comment (svedova) [inline .tsconfig:35]: **suggestion** Shall we add a newline to the end of the file? That's usually a default.

File: .tsconfig [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (svedova) [inline .tsconfig:1]: **question** Is this configuration the default one suggested by typescript? If not, do you know if is there any?

**Review by svedova — COMMENTED** [review]
Thanks @ivanlori for this. Left a suggestion and a question :)

File: .tsconfig (hunk 2 above)
Comment (ivanlori) [reply .tsconfig:1]: The only reference that I found is this -> https://www.typescriptlang.org/docs/handbook/tsconfig-json.html , it gives us different type of basic configurations based on the node version of the project.

Comment (svedova) [reply .tsconfig:1]: I guess it points to this repository: https://github.com/tsconfig/bases/. It's the one you mean right?

File: .tsconfig (hunk 1 above)
Comment (ivanlori) [reply .tsconfig:35]: I don't use to add a newline at the end, but we can enable by default on eslint if I am not wrong.

File: .tsconfig (hunk 2 above)
Comment (svedova) [reply .tsconfig:1]: I think it's fine the configuration, just wondered where we got these from!

File: .tsconfig (hunk 1 above)
Comment (svedova) [reply .tsconfig:35]: Let's do it, I think it's a practice coming from the past. https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file :)

File: .tsconfig (hunk 2 above)
Comment (ivanlori) [reply .tsconfig:1]: do you mean my configuration or the ones from the documentation?

Comment (svedova) [reply .tsconfig:1]: the one in this MR is fine :)

File: .tsconfig (hunk 1 above)
Comment (ivanlori) [reply .tsconfig:35]: it should work now, I added rule: "newLine": "LF", and added a empty line at the end of the file. Is it what we do?

Comment (svedova) [reply .tsconfig:35]: Yup! https://github.com/stormkit-io/app-stormkit-io/pull/18 seems to be similar to this branch. Shall we close this PR or they are different?

Comment (ivanlori) [reply .tsconfig:35]: same branch but different PR

**Pull request closed without merging**